packages/genesys-spark-components/src/components/beta/gux-icon-tooltip/gux-icon-tooltip.scss

```diff
@@ -0,0 +1,28 @@
+@use '~genesys-spark/src/style/focus.scss';
+@use '~genesys-spark/src/style/mixins.scss';
+
+.gux-icon-tooltip {
+  padding: 0;
+  background-color: transparent;
+  border: none;
+  border-radius: var(--gse-ui-formControl-label-tooltip-focusRing-borderRadius);
+
+  &:focus-visible {
+    @include focus.gux-focus-ring-small;
+
+    outline-offset: var(--gse-semantic-focusRing-offset);
+  }
```

Comment:
I think there should be a `border-radius` here when focusing on the `icon-tooltip`

Reply:
Should work with the next build.